session/session.go

```diff
@@ -141,3 +228,25 @@ func (s *Session) readFromHandler() map[string]any {
 	}
 	return data
 }
+
+func (s *Session) ageFlashData() {
+	oldAny := s.Get("_flash.old", make([]any, 0)).([]any)
+	old := make([]string, len(oldAny))
+	for i, v := range oldAny {
+		old[i] = v.(string)
+	}
+
+	s.Forget(old...)
+	s.Put("_flash.old", s.Get("_flash.new", make([]any, 0)))
+	s.Put("_flash.new", make([]any, 0))
+}
```

Comment:
Sorry, I didn't get the idea of this function, could you please make an example?

Reply:
In this PR, I am adding a feature to manage flash data. For which I have added a `Flash` method which will add data for the next request in the session. So I have added two arrays: one is `_flash.new` (in attributes) which will store flash keys of the current request and `_flash.old` will store flash keys from the previous request. To achieve this when saving, I don't want to save data (`Save`) from the previous request (i.e., keys stored in `_flash.old`). I will remove those keys, and whatever we have in the current request will be added into `_flash.old` so that it can be removed in the next request.

![image](https://github.com/goravel/framework/assets/84431594/af76c859-15cd-47d7-ae25-acea171eb4a2)
![image](https://github.com/goravel/framework/assets/84431594/8de4b8d7-0fad-49f7-83ee-40a2c7c6170e)
![image](https://github.com/goravel/framework/assets/84431594/f8978c45-b1ef-4728-84eb-9034fe4dcdda)

Reply:
Okay, one more question, if we add the `flash` method, is it better to add the other methods: `reflash`, `keep`, and  `now`?

Reply:
Good suggestion, I will add other three methods.